src/main/java/seedu/classes/Storage.java

```diff
@@ -0,0 +1,41 @@
+package seedu.classes;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+    private static final String PASSWORD_FILE_PATH = "./password.txt";
+    private int passwordHash = 0;
+
+    public Storage(Ui ui) {
+        try {
+            if (!new File(PASSWORD_FILE_PATH).exists()) {
+                new File(PASSWORD_FILE_PATH).createNewFile();
+                FileWriter fw = new FileWriter(PASSWORD_FILE_PATH);
+                fw.write(new Login().createNewUser(ui));
+                fw.close();
+            }
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        this.passwordHash = getPasswordHashFromStorage();
```

Comment:
Maybe its better to have a guard clause to prevent arrowhead. 
If it exists call `getPasswordHashFromStorage` and return. Then, use try-catch afterwards and initialise passwordHash with `Login().createNewUser`. directly.

Reply:
Editted and pushed